fullResults.txt

```diff
@@ -0,0 +1 @@
CertificationId":"04072014-2607-1","classificationType":{"name":"Modular EHR","id":1},"otherAcb":null,"certificationStatus":{"name":"Active","id":1},"developer":{"name":"EMRConnect","id":402},"product":{"versionId":5234,"name":"Access Profiler Enterprise","id":626,"version":"V6.1"},"certificationEdition":{"name":"2014","id":2},"practiceType":{"name":"Ambulatory","id":1},"certifyingBody":{"name":"Drummond Group","id":3},"certificationDate":1396843200000,"ics":null,"sedTesting":null,"qmsTesting":null,"accessibilityCertified":null,"productAdditionalSoftware":null,"transparencyAttestation":"Affirmative","transparencyAttestationUrl":"http://emrconnect.com/products.html","countCerts":3,"countCqms":0,"countCorrectiveActionPlans":0,"countCurrentCorrectiveActionPlans":0,"countClosedCorrectiveActionPlans":0},{"id":5043,"testingLabId":null,"testingLabName":null,"chplProductNumber":"CHP-023885","reportFileLocation":"http://www.drummondgroup.com/images/ehr_pdf/EMRConnect_APE_V6.1_Amb_08282014-2609-1.pdf","sedReportFileLocation":null,"sedIntendedUserDescription":null,"sedTestingEnd":null,"acbCertificationId":"08282014-2609-1","classificationType":{"name":"Modular EHR","id":1},"otherAcb":null,"certificationStatus":{"name":"Active","id":1},"developer":{"name":"EMRConnect","id":402},"product":{"versionId":5234,"name":"Access Profiler Enterprise","id":626,"version":"V6.1"},"certificationEdition":{"name":"2014","id":2},"practiceType":{"name":"Ambulatory","id":1},"certifyingBody":{"name":"Drummond Group","id":3},"certificationDate":1409198400000,"ics":null,"sedTesting":null,"qmsTesting":null,"accessibilityCertified":null,"productAdditionalSoftware":null,"transparencyAttestation":"Affirmative","transparencyAttestationUrl":"http://emrconnect.com/products.html","countCerts":4,"countCqms":0,"countCorrectiveActionPlans":0,"countCurrentCorrectiveActionPlans":0,"countClosedCorrectiveActionPlans":0},{"id":5042,"testingLabId":null,"testingLabName":null,"chplProductNumber":"CHP-023884","reportFileLocation":"http://www.drummondgroup.com/images/ehr_pdf/EMRConnect_APE_V6.1_Inp_08282014-2610-1.pdf","sedReportFileLocation":null,"sedIntendedUserDescription":null,"sedTestingEnd":null,"acbCertificationId":"08282014-2610-1","classificationType":{"name":"Modular EHR","id":1},"otherAcb":null,"certificationStatus":{"name":"Active","id":1},"developer":{"name":"EMRConnect","id":402},"product":{"versionId":5234,"name":"Access Profiler Enterprise","id":626,"version":"V6.1"},"certificationEdition":{"name":"2014","id":2},"practiceType":{"name":"Inpatient","id":2},"certifyingBody":{"name":"Drummond Group","id":3},"certificationDate":1409198400000,"ics":null,"sedTesting":null,"qmsTesting":null,"accessibilityCertified":null,"productAdditionalSoftware":null,"transparencyAttestation":"Affirmative","transparencyAttestationUrl":"http://emrconnect.com/products.html","countCerts":4,"countCqms":0,"countCorrectiveActionPlans":0,"countCurrentCorrectiveActionPlans":0,"countClosedCorrectiveActionPlans":0}]}
```

Comment:
Is this an output file? Is it what's generated when the sample application is run? If so, it shouldn't be committed to the system, and should probably be in the .gitignore file. It falls into the category of "build" objects, and doesn't need to be tracked by git